tools/UpdatePaths/Scripts/21681_areas.txt

```diff
@@ -0,0 +1,179 @@
+/area/hallway/secondary/construction : /area/station/public/construction
+/area/construction : /area/station/public/construction
+
+
+# ----- Deleted areas! -----
+/area/crew_quarters/recruit : @DELETE
```

Comment:
we don't actually use these anywhere so it's fine but i don't think you want to perform a delete update on an /area path, that would either cause the tile to have no area, or to (maybe) default to `/area`. i think it's fine not to mention these at all in the update paths, and let their existence in the map be a compilation error so it fails loudly

Reply:
Good idea, I had it originally put in a conflict marker (but that was buggy, so I just made it delete), but just letting them stay is probably better